### PR TITLE
Feat: calculate taxi cost

### LINKED ISSUE #55: Taxi Cost

### PR DESCRIPTION
**Pull request by AngeloTadeucci**
- Resolves #55 

Need to wait for these pull requests to get merged
- https://github.com/AngeloTadeucci/Maple2.File/pull/19
- https://github.com/kOchirasu/Maple2.Lua/pull/3

### CODE REVIEW

File: Maple2.Server.Game/PacketHandlers/TaxiHandler.cs [hunk 1]
```diff
@@ -55,40 +58,89 @@ private enum Command : byte {
...
     }
 
-    private static void HandleMeretAirTaxi(GameSession session, IByteReader packet) {
+    private void HandleMeretAirTaxi(GameSession session, IByteReader packet) {
         int mapId = packet.ReadInt();
+        if (!CheckMapCashCall(session, mapId, MapMetadata)) {
+            return;
+        }
+
+        const int meretTaxiPrice = 15;
```
Comment (Zintixx) [inline Maple2.Server.Game/PacketHandlers/TaxiHandler.cs:135]: add this to constants

File: Maple2.Server.Tests/Usings.cs [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿global using NUnit.Framework;
```
Comment (Zintixx) [inline Maple2.Server.Tests/Usings.cs:1]: whats this

Comment (AngeloTadeucci) [reply Maple2.Server.Tests/Usings.cs:1]: global using for all tests

Comment (Zintixx) [reply Maple2.Server.Tests/Usings.cs:1]: oh right i forgot that's a thing

**Review by Zintixx — APPROVED** [review]

**Pull request merged**